### ISSUE TITLE
add Header only in the first page

### ISSUE THREAD

**Issue by elvishuges** [motla/vue-document-editor#21]
is there a way to add header only in the first page. and keep the next pages with normal headers space ?

currently, when in set the header space using page_margins variable it is aapplied to all page.

**Comment by motla** [comment]
Currently it is not possible, but we can implement it. The good workflow would be to accept also a Function for `page_margins`, the same way it is done for `overlay`.
If you want to try implementing it yourself and make a pull request you are welcome to do so. Otherwise can look at it but probably not before the end of this month...

**Comment by elvishuges** [comment]
greate man. but I don't understand your code enough to implement this similar to overlay. can you help me in this issues ?

**Comment by motla** [comment]
Yes, you only have to modify the DocumentEditor.vue file.
1. First you have to allow a Function to be passed to the `page_margins` property ([see docs here](https://vuejs.org/guide/components/props.html#prop-validation))
So you just have to set `type: [String, Function]` instead of `type: String` at:

https://github.com/motla/vue-document-editor/blob/03fea9fce060d5cbc1ef91177dd97e548ae2ef8b/src/DocumentEditor/DocumentEditor.vue#L60

2. Next search for `this.page_margins` calls in the DocumentEditor.vue file (there should be two).
For now it only accepts a string, which is directly put in CSS `padding` attribute of the pages responsible for their displayed margins.
But now what we want to do is check if the user provided a Function, call it with two arguments: the current page number and the total number of pages, just like the overlay function (i.e. `page_margins(page, total)`). This function written by the user should simply return a string depending of its arguments.

**Comment by motla** [comment]
Hi @elvishuges 
I just released v2.2.0 (Vue3) and v1.4.0 (Vue2) including your feature request.
You can now specify a method for `page_margins`.
For example, to get a larger header space only on first page:
```javascript
  page_margins: (index) => (index == 0 ? "30mm 15mm 10mm 15mm" : "10mm 15mm"),
//                                         ↑    →    ↓    ←        ↕︎    ↔︎
```
Have fun!
